src/__tests__/vue-router.js

```diff
@@ -1,26 +1,38 @@
-// Please notice that this example is a draft example on how to test
-// the router.
-// Related issue on Vue Test Utils: https://github.com/vuejs/vue-test-utils-next/issues/152
+/* eslint-disable vue/require-prop-types */
+/* eslint-disable vue/one-component-per-file */
 
 import '@testing-library/jest-dom'
 import {waitFor} from '@testing-library/dom'
+import {defineComponent} from 'vue'
+import {createRouter, createWebHistory} from 'vue-router'
 import {render, fireEvent} from '..'
 import App from './components/Router/App.vue'
 import Home from './components/Router/Home.vue'
 import About from './components/Router/About.vue'
 
-const routes = [
-  {path: '/', component: Home},
-  {path: '/about', component: About},
-]
+test('full app rendering/navigating from base URL', async () => {
+  // Create a Router instance
+  // https://next.router.vuejs.org/api/#createrouter
+  // using a HTML5 history.
+  // https://next.router.vuejs.org/api/#createwebhistory
+  const router = createRouter({
+    history: createWebHistory(),
+    routes: [
+      {path: '/', component: Home},
+      {path: '/about', component: About},
+    ],
+  })
 
-test('full app rendering/navigating', async () => {
-  // Notice how we pass a `routes` object to our render function.
-  const {findByText, getByText, getByTestId} = render(App, {routes})
+  const {findByText, getByText, getByTestId} = render(App, {
+    global: {
+      plugins: [router],
+    },
+  })
 
   // Vue Router navigation is async, so we need to wait until the
   // initial render happens
   expect(await findByText('You are home')).toBeInTheDocument()
+  expect(getByTestId('location-display')).toHaveTextContent('/')
```

Comment:
Out of curiosity, any reason for checking the location display instead of checking the path on `window.location.href`?

Reply:
Hmm not really, we simply added an element that renders the route to the DOM so we don't suggest checking against "implementation details" such as window.location.href ;)

Reply:
> Although I've contributed a little, I haven't seen the entire codebase. So some curious questions will pop up here and there. 😅

Thank you for your thoughts!

Reply:
Fair enough :smirk: